relays/bin-substrate/src/cli/relay_headers_and_messages/mod.rs

```diff
@@ -317,8 +318,9 @@ where
 		}
 
 		// start on-demand header relays
+		let metrics_params = self.base().common().metrics_params.clone();
 		let (left_to_right_on_demand_headers, right_to_left_on_demand_headers) =
-			self.mut_base().start_on_demand_headers_relayers().await?;
+			self.mut_base().start_on_demand_headers_relayers(&metrics_params).await?;
```

Comment:
Do we need to send this as a parameter to the function ? I think we can access `self.common.metrics_params` directly in `start_on_demand_headers_relayers()`.

Reply:
Right, thank you!